jobs/models.py

```diff
@@ -11,13 +11,14 @@
 
 @python_2_unicode_compatible
 class Job(models.Model):
-    title = models.CharField(max_length=255, verbose_name='Başlık')
-    company = models.CharField(max_length=255, verbose_name='Şirket Adı')
-    url = models.URLField(max_length=255, verbose_name='Başvuru Linki')
+    title = models.CharField(verbose_name='Başlık', max_length=255)
```

Comment:
Bu verbose_name değişikliklerini ayrı bir pull request'te tartışabiliriz ama #51ile alakasız olduğu için şimdilik kaldıralım.